lib/composer.js

```diff
@@ -100,6 +99,10 @@ function generateJsonReport(
   return data;
 }
 
+function getDefaultProjectName() {
+  return process.cwd().split(path.sep).pop();
```

Comment:
this should use also uses the `root` parameter passed to `inspect()`, as it's relative to the `cwd`